controller_manager/src/ros2_control_node.cpp

```diff
@@ -32,24 +32,29 @@ int main(int argc, char ** argv)
     executor,
     manager_node_name);
 
-  // load controller_manager update time parameter
-  int update_rate = 100;
-  if (!cm->get_parameter("update_rate", update_rate)) {
-    throw std::runtime_error("update_rate parameter not existing or empty");
-  }
-  RCLCPP_INFO(cm->get_logger(), "update rate is %d Hz", update_rate);
-
-  auto timer = cm->create_wall_timer(
-    std::chrono::milliseconds(1000 / update_rate),
-    [&cm]() {
-      cm->read();
-      cm->update();
-      cm->write();
-    },
-    cm->deterministic_callback_group_);
+  // TODO: Due to issues with the MutliThreadedExecutor, this control loop does not rely on the executor. When the
```

Comment:
This should restore the CI

```suggestion
  // TODO(anyone): Due to issues with the MutliThreadedExecutor, this control loop does not rely on the executor (see issue #260).
```

Reply:
Done